### DISCUSSION TITLE
Use ReportGenerator in macOS

### DISCUSSION THREAD

**Discussion by jiguaita** [danielpalme/ReportGenerator#478] [Q&A] [answered]
Hi, i can use ReportGenerator in macOS? Thanks.

**Comment by danielpalme** [comment, marked as answer]
Yes. You can use the [.NET global tool](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools#install-a-global-tool):
https://www.nuget.org/packages/dotnet-reportgenerator-globaltool

**Reply by jiguaita** [reply]
Hi, i try install the tool with nugget manager in my test project but i obtain this error: Package 'dotnet-reportgenerator-globaltool 5.0.2' has a package type 'DotnetTool' that is not supported by project

I use Net 5

Do you know the problem?

Thanks.

**Reply by danielpalme** [reply]
It seems as if you are trying to install the global tool in your project.
This is not how global tools work. As the name "global" states, the tool is installed globally on you machine.

So first you install the tool with this command:
```
dotnet tool install --global dotnet-reportgenerator-globaltool --version 5.0.2
```

Then you can execute it with this command:
```
reportgenerator [options]
```


See also this issue: https://github.com/danielpalme/ReportGenerator/issues/315
Or the documentation on global tools: https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools#install-a-global-tool
